### PR TITLE
[Keyboard] NK Classic TKL

### PR DESCRIPTION
**Pull request by yiancar**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/novelkeys/nk_classic_tkl/info.json [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+            {"matrix": [5, 14], "x": 195, "y": 64, "flags": 1},
+            {"matrix": [5, 13], "x": 176, "y": 64, "flags": 1},
+            {"matrix": [5, 12], "x": 160, "y": 64, "flags": 1},
+            {"matrix": [5, 11], "x": 144, "y": 64, "flags": 1},
+            {"matrix": [5, 6], "x": 90, "y": 64, "flags": 4},
+            {"matrix": [5, 2], "x": 35, "y": 64, "flags": 1},
+            {"matrix": [5, 1], "x": 19, "y": 64, "flags": 1},
+            {"matrix": [5, 0], "x": 3, "y": 64, "flags": 1}
+        ],
+        "led_count": 92,
```
Comment (zvecr) [inline keyboards/novelkeys/nk_classic_tkl/info.json:164]:
```suggestion
```

File: keyboards/novelkeys/nk_classic_tkl/info.json [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+    "features": {
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+        "rgblight": false
```
Comment (zvecr) [inline keyboards/novelkeys/nk_classic_tkl/info.json:16]:
```suggestion
        "bootmagic": true,
        "command": false,
        "console": false,
        "extrakey": true,
        "mousekey": true,
        "nkro": true,
        "rgb_matrix": true
```

File: keyboards/novelkeys/nk_classic_tkl/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+enum my_keycodes {
+  RETRO_RGB = QK_KB_0
```
Comment (zvecr) [inline keyboards/novelkeys/nk_classic_tkl/keymaps/default/keymap.c:19]:
```suggestion
  RETRO_RGB = QK_USER_0
```

File: keyboards/novelkeys/nk_classic_tkl/readme.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# NK Classic TKL
+
+This is a TKL PCB. It supports VIA and full per-key RGB.
+
+* Keyboard Maintainer: [Yiancar](https://yiancar-designs.com/) and on [GitHub](https://github.com/yiancar)
+* Hardware Supported: A 65% with macro keys keyboard with STM32F072CB or APM compatible
```
Comment (zvecr) [inline keyboards/novelkeys/nk_classic_tkl/readme.md:6]: 65% tkl?

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/novelkeys/nk_classic_tkl/info.json (hunk 1 above)
Comment (yiancar) [reply keyboards/novelkeys/nk_classic_tkl/info.json:162]: nice! this is no longer needed!

File: keyboards/novelkeys/nk_classic_tkl/readme.md (hunk 4 above)
Comment (yiancar) [reply keyboards/novelkeys/nk_classic_tkl/readme.md]: Upsie copypasta! thank you:)

File: keyboards/novelkeys/nk_classic_tkl/readme.md
Comment (dunk2k) [inline keyboards/novelkeys/nk_classic_tkl/readme.md:1]: Image of PCB in question would be helpful 🙂

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Review by keyboard-magpie — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**